playbooks/idoverridegroup/idoverridegroup-present.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Idoverridegroup present example
+  hosts: ipaserver
+  become: no
+
+  tasks:
+  - name: Ensure idoverridegroup NAME is present
+    ipaidoverridegroup:
+      ipaadmin_password: SomeADMINpassword
+      idview: test_idview
+      anchor: test_group
```

Comment:
It would be nice to have `gid` in the example, in case DOC wants to use it as an example.